custom-basic-authenticator-for-ForcePasswordOTP/pom.xml

```diff
@@ -42,7 +42,12 @@
         <dependency>
```

Comment:
What is the reason for changing the folder name? we usually keep them lower case.

Reply:
Renamed.